docs/samples/Microsoft.ML.Samples/Dynamic/WordEmbeddingTransform.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Text;
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class WordEmbeddingTransform
+    {
+        public static void ExtractEmbeddings()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert to IDataView.
+            var data = SamplesUtils.DatasetUtils.GetSentimentData();
+            var trainData = ml.Data.ReadFromEnumerable(data);
+
+            // Preview of the data.
+            //
+            // Sentiment    SentimentText
+            // true         Best game I've ever played.
+            // false        ==RUDE== Dude, 2.
+            // true          Until the next game, this is the best Xbox game!
+
+            // Pipeline which goes through SentimentText and normalize it, tokenize it by words, and removes default stopwords.
+            // After all that cleaning, we apply WordEmbedding with GloVeTwitter25D.
+            // 25D means each word mapped into 25 dimensional space, basically each word represented by 25 float values.
+            var wordsPipeline = ml.Transforms.Text.NormalizeText("NormalizedText", "SentimentText", keepDiacritics: false, keepPunctuations: false, keepNumbers: false)
+                .Append(ml.Transforms.Text.TokenizeWords("Words", "NormalizedText"))
+                .Append(ml.Transforms.Text.RemoveDefaultStopWords("CleanWords", "Words"))
+                .Append(ml.Transforms.Text.ExtractWordEmbeddings("WordEmbeddings", "CleanWords", WordEmbeddingsExtractingTransformer.PretrainedModelKind.GloVeTwitter25D));
+
+            var embeddingDataview = wordsPipeline.Fit(trainData).Transform(trainData);
+            // Small helper to print the text inside the columns, in the console. 
+
+            // Preview of the CleanWords column obtained after processing SentimentText.
+            var cleanWords = embeddingDataview.GetColumn<VBuffer<ReadOnlyMemory<char>>>(ml, "CleanWords");
+            Console.WriteLine($" CleanWords column obtained post-transformation.");
+            foreach (var featureRow in cleanWords)
+            {
+                foreach (var value in featureRow.GetValues())
+                    Console.Write($"{value} ");
+                Console.WriteLine("");
+            }
+
+            Console.WriteLine("===================================================");
+            // best game ive played.
```

Comment:
do you want this here? feel like the less "stuff" we have, the better.. likewise below.

Reply:
I'm actually want to extend this sample with using custom model, in which I can specify 2 or 3 dimensions for words best game played and dude and us it to show how output looks like. Much better to show 9 floats than 75. And I think it would be nice to show up words after cleaning.

---
In reply to: [252540489](https://github.com/dotnet/machinelearning/pull/2339#discussion_r252540489) [](ancestors = 252540489)